targets/TARGET_Cypress/TARGET_PSOC6/PeripheralPins.h

```diff
@@ -68,4 +68,13 @@ extern const PinMap PinMap_ADC[];
 extern const PinMap PinMap_DAC[];
 #endif
 
+#if DEVICE_QSPI
```

Comment:
This should be combined with above (line 55-59)

Reply:
This is addressed by https://github.com/ARMmbed/mbed-os/pull/10692/commits/bab34cb4675ced134deaae178440c20fb9cd9779

Note: DEVICE_QSPI is not yet enabled for Cypress targets (cy_qspi_api.c will not compile when device_has: "QSPI" added to targets.json). This will be addressed in subsequent PR.

Reply:
ok